mhkit/power/quality/calc_alpha0_freq.m

```diff
@@ -0,0 +1,87 @@
+function [alpha0,freq] = calc_alpha0_freq(u_m,method,methodopts)

```

Comment:
I would change this to be called "calc_fundamental_freq" for clarity. Alpha0 can me other things in different applications